.travis.yml

```diff
@@ -164,6 +164,10 @@ matrix:
         - npm install -g @zeus-ci/cli
         - zeus upload -t "application/zip+wheel" py/dist/*
 
+  allow_failures:
+    - env: SUITE=format-check
+    - env: SUITE=lint
```

Comment:
This is not the reason the build fails. If we ignore formatting and linters here too, noone will ever pay attention to them and fixing them will be a huge pain since we just postpone fixing them.

Reply:
@jan-auer lint does not work for the same reason lint does not work in sentry-rust: im requires unstable annotations. I just made the same change as you did in sentry-rust.